File: src/main/java/CalculatorController.java
import helpers.SaverSingleton;

import javax.servlet.ServletException;
import javax.servlet.annotation.WebServlet;
import javax.servlet.http.HttpServlet;
import javax.servlet.http.HttpServletRequest;
import javax.servlet.http.HttpServletResponse;
import java.io.IOException;

@WebServlet(name = "calculator", value = "/calculator.jsp")
public class CalculatorController extends HttpServlet {

    @Override
    protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
        req.setAttribute("first", SaverSingleton.getInstance().getFirst());
        req.setAttribute("second", SaverSingleton.getInstance().getSecond());
        req.getRequestDispatcher("/index.jsp").forward(req, resp);
    }

    @Override
    protected void doPost(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
        req.getRequestDispatcher("/index.jsp").forward(req, resp);
    }
}

File: src/main/kotlin/com/elistratov/CalculatorController.kt
package com.elistratov

import java.nio.file.Paths
import javax.servlet.annotation.WebServlet
import javax.servlet.http.HttpServlet
import javax.servlet.http.HttpServletRequest
import javax.servlet.http.HttpServletResponse

@WebServlet(name = "calcKT", value = "/calc")
class CalculatorController : HttpServlet() {
    override fun doGet(req: HttpServletRequest?, resp: HttpServletResponse?) {
       servletContext
               .getRequestDispatcher("/calculator.jsp")
               .forward(req,resp)
        System.out.println(Paths.get("calculator.jsp").toAbsolutePath())
    }
}